kits/arms/config/A-2099-07.cfg.yaml

```diff
@@ -1,7 +1,7 @@
-# T-Series 6-DoF Arm with Gripper
+# X-Series 7-DoF Double Shoulder Arm
 version: 1.0
 families: ["Arm"]
-names: ["J1_base", "J2_shoulder1A", "J3_shoulder2", "J4_elbow1", "J5_elbow2", "J6_wrist1", "J7_wrist2"]
+names: ["J1_base", J2A_shoulder1", "J3_shoulder2", "J4_elbow1", "J5_elbow2", "J6_wrist1", "J7_wrist2"]
```

Comment:
Did the naming convention change at some point? I assume the one from the config repo is correct?